core/arch/arm/sm/sm_a32.S

```diff
@@ -216,6 +216,28 @@ FUNC sm_init , :
 	mrs	r1, cpsr
 	cps	#CPSR_MODE_MON
 	mov	sp, r0
+
```

Comment:
This looks like a normal world problem, it doesn't have to be handled in OP-TEE.